portfolio/.eslintcache

```diff
@@ -0,0 +1 @@
>\n          <a className=\"navbar__link\" href=\"\">\n            Stories,\n          </a>\n        </li>\n        <li className=\"navbar__item\">\n          <a className=\"navbar__link\" href=\"#\">\n            About\n          </a>\n        </li>\n      </ul>\n    </nav>\n  );\n}\n","/Users/maja/Desktop/Projects/Photography_portfolio/portfolio/src/components/Carousel.js",["50","51"],"/Users/maja/Desktop/Projects/Photography_portfolio/portfolio/src/components/Heading.js",[],"/Users/maja/Desktop/Projects/Photography_portfolio/portfolio/src/components/GalleryDisplayButtons.js",[],"/Users/maja/Desktop/Projects/Photography_portfolio/portfolio/src/components/Form.js",["52","53"],"/Users/maja/Desktop/Projects/Photography_portfolio/portfolio/src/components/Gallery.js",[],"/Users/maja/Desktop/Projects/Photography_portfolio/portfolio/src/components/Footer.js",[],{"ruleId":"54","replacedBy":"55"},{"ruleId":"56","replacedBy":"57"},{"ruleId":"58","severity":1,"message":"59","line":82,"column":17,"nodeType":"60","messageId":"61","endLine":82,"endColumn":23},{"ruleId":"62","severity":1,"message":"63","line":8,"column":11,"nodeType":"64","endLine":8,"endColumn":48},{"ruleId":"62","severity":1,"message":"63","line":13,"column":11,"nodeType":"64","endLine":13,"endColumn":47},{"ruleId":"62","severity":1,"message":"63","line":18,"column":11,"nodeType":"64","endLine":18,"endColumn":48},{"ruleId":"58","severity":1,"message":"65","line":22,"column":9,"nodeType":"60","messageId":"61","endLine":22,"endColumn":17},{"ruleId":"66","severity":1,"message":"67","line":59,"column":6,"nodeType":"68","endLine":59,"endColumn":8,"suggestions":"69"},{"ruleId":"58","severity":1,"message":"70","line":8,"column":10,"nodeType":"60","messageId":"61","endLine":8,"endColumn":13},{"ruleId":"58","severity":1,"message":"71","line":12,"column":9,"nodeType":"60","messageId":"61","endLine":12,"endColumn":24},"no-native-reassign",["72"],"no-negated-in-lhs",["73"],"no-unused-vars","'object' is assigned a value but never used.","Identifier","unusedVar","jsx-a11y/anchor-is-valid","The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md","JSXOpeningElement","'interval' is assigned a value but never used.","react-hooks/exhaustive-deps","React Hook useEffect has missing dependencies: 'moveToNextSlide' and 'slideContainer'. Either include them or remove the dependency array.","ArrayExpression",["74"],"'url' is assigned a value but never used.","'handleFileInput' is assigned a value but never used.","no-global-assign","no-unsafe-negation",{"desc":"75","fix":"76"},"Update the dependencies array to be: [moveToNextSlide, slideContainer]",{"range":"77","text":"78"},[1942,1944],"[moveToNextSlide, slideContainer]"]
```

Comment:
Hey @majac91 this file doesn't need to be on the repo. Delete this file and commit it. then, add it to the gitignore.